screens/WorkoutDetails.js

```diff
@@ -67,9 +67,9 @@ class WorkoutDetailsScreen extends React.Component {
                         </View>
                         <View style={styles.exercises}>
                             {
-                                this.workout.exercises.map((exercise, ei) => {
+                                this.workout.exercises.map((exercise, id) => {
```

Comment:
Just to make sure, second parameter 'ei' is just an index (starting from 0 to N), it's not an ID of the element. Don't know if you need an id or just a number in a sequence.